packages/Webkul/Admin/tests/Feature/Catalog/AttributeFamilyTest.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+use Webkul\Attribute\Models\AttributeFamily as AttributeFamilyModel;
+
+use function Pest\Laravel\deleteJson;
+use function Pest\Laravel\get;
+use function Pest\Laravel\getJson;
+use function Pest\Laravel\postJson;
+use function Pest\Laravel\putJson;
+
+afterEach(function () {
+    /**
+     * Clean attribute families, excluding ID 1 (i.e., the default attribut family). A fresh instance will always have ID 1.
+     */
+    AttributeFamilyModel::query()
+        ->whereNot('id', 1)
+        ->delete();
+});
+
+it('should return attribute family listing page', function () {
+    // Act and Assert
+    $this->loginAsAdmin();
+
+    get(route('admin.catalog.families.index'))
+        ->assertOk()
+        ->assertSeeText(trans('admin::app.catalog.families.index.title'))
+        ->assertSeeText(trans('admin::app.catalog.families.index.add'));
+});
+
+it('should return listing items of attribute family', function () {
+    // Act and Assert
```

Comment:
Use attribute family factory and create one more attribute family via factory. 

And remove the hard-coded assertion.

Reply:
done